syncd/scripts/syncd_init_common.sh

```diff
@@ -119,74 +119,175 @@ config_syncd_cisco_8000()
     fi
 }
 
-config_syncd_bcm()
+function merge_config_bcm_files()
 {
+    to_file=$1
+    from_file=$2
+    message=$3
+    override=false
+    echo "" >> $to_file
+    echo "# Start of $message" >> $to_file
+    while read line
+    do
+        line=$( echo $line | xargs )
+        if [ ! -z "$line" ];then
+            if [ "${line::1}" == '#' ];then
+                echo $line >> $to_file
+            elif [ "$line" == "[Overwrite Section]" ];then
```

Comment:
If there is an extra space at end, no match will happen, right? How to consider this scenario?

Reply:
Yes, the script will skip empty lines